lib/util.js

```diff
@@ -136,6 +137,28 @@ util = {
 
             callback(null, data);
         });
+    },
+
+    /**
+     * Writes a JSON blob to the given path.
+     *  * If ``options.pretty`` is true, output will be pretty printed. (Default false)
+     *  * If ``options.overwrite`` is false, output file will be overwritten (Default true)
+     *
+     * @param path
+     * @param data
+     * @param options
+     * @param callback
+     */
+    writeJSON: function (path, data, options, callback) {
+        var json = (options.pretty) ? JSON.stringify(data, null, 4) : JSON.stringify(data),
+            flag = (options.overwrite) ? 'w' : 'wx';
+
+        fs.writeFile(path, json, {flag: flag}, function (err) {
```

Comment:
we can keep these two stored on top as 

```javascript 
FSWF_FLAG_W = { flag: 'w' };
FSWF_FLAG_WX = { flag: 'wx' };
```

... and then use later in this function

```javascript 
fs.writeFile(path, json, overwrite ? FSWF_FLAG_WX : FSWF_FLAG_W, function (err) {});
```